woocommerce/woocommerce-android
# Fix FeedbackPrefs initialization on UI Tests

### Description
This PR fixes the UI tests initialization, after we introduced Hilt to the code base, all the app's initialization has been moved to the AppInitializer class, but I missed the FeedbackPrefs init method.
I moved it to it in 09554cf, to make sure it gets called when creating tests.

Something else that I noticed, is that if I try to test a class that has multiple tests (the case of `ScreenshotTest` now) at the same time, the second test fails, this is because we try to initialize the app for each test individually using the `InitializationRule` (same behavior of Robolectric before), so this creates issues for components that don't expect to be initialized multiple times (from my testing this occurs only for ZendeskHelper). So to fix this, I moved the `mocksDemo` to its own class in 123a5a4. @designsimply if you need to have multiple tests in the same class, then we can try to find another solution, just let me know.

### Testing instructions
Run the test of MocksTest class, and confirm that you can navigate to products screen.
